packages/venia-ui/lib/components/Newsletter/newsletter.js

```diff
@@ -0,0 +1,95 @@
+import React, { Fragment, useEffect } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Form } from 'informed';
+import { shape, string } from 'prop-types';
+
+import { useNewsletter } from '@magento/peregrine/lib/talons/Newsletter/useNewsletter';
+import { useToasts } from '@magento/peregrine';
+
+import { isRequired } from '../../util/formValidators';
+import { mergeClasses } from '../../classify';
+import FormError from '../FormError';
+import Button from '../Button';
+import LoadingIndicator from '../LoadingIndicator';
+import TextInput from '../TextInput';
+import defaultClasses from './newsletter.css';
+
+const Newsletter = props => {
+    const { formatMessage } = useIntl();
+    const classes = mergeClasses(defaultClasses, props.classes);
```

Comment:
@shikhamis11 Sorry, I missed this one in the first review. `mergeClasses` should be changed for `useStyle`.
```
import { useStyle } from '../../classify';
// ...
const classes = useStyle(defaultClasses, props.classes);
```

Reply:
sure